client/public/samples/sample1.json

```diff
@@ -0,0 +1,94 @@
+{
+  "collectionType": "preprint",
+  "branding": {
+    "colors": {
+      "primary": "#215264",
+      "secondary": "#eeeeee",
+      "background": "#e14b5a",
+      "backgroundText": "#ffffff"
+    },
+    "logo": {
+      "label": "socarxiv",
+      "url": "https://osf.io/preprints/assets/img/provider_logos/socarxiv-small-bb26fbaa227c1fc8b5cc6f8c7d1705e69c18bfd9.png",
+      "height": "140px"
+    }
+  },
+  "layout": [{
+    "type": "landing-hero",
+    "title": "You can override collection title here",
+    "tagline": "Open archive of the social sciences",
+    "data": null,
+    "settings": {
+      "useLogo": true
+    }
+  }, {
+    "type": "landing-list",
+    "title": "Subjects",
+    "data": "subjects",
+    "settings": null
+  }, {
+    "type": "landing-board",
+    "title": "Steering Committee",
+    "data": "people",
+    "settings": null
+  }, {
+    "type": "landing-copyright",
+    "title": null,
+    "data": "links",
+    "settings": null
+  }],
+  "data": {
+    "subjects": [{
+      "name": "Arts and Humanities",
+      "label": "Fine Arts, History, Music, Philosophy, Religion",
+      "link": "/preprints/socarxiv/discover?subject=Arts%20and%20Humanities"
+    }, {
+      "name": "Arts and Humanities",
```

Comment:
If this is meant to be used as-is for demos, I would suggest having four different subjects here like SocArXiv does: https://staging.osf.io/preprints/socarxiv